frontends/web/src/components/status/status.tsx

```diff
@@ -15,98 +15,71 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
-import { apiGet, apiPost } from '../../utils/request';
+import { ReactNode, useCallback, useEffect, useState } from 'react';
 import { CloseXWhite } from '../icon';
 import style from './status.module.css';
+import { getConfig } from '../../api/backend';
+import { setConfig } from '../../api/backend';
+
 
-interface State {
-    show: boolean;
-}
 
-interface StatusProps {
+type TPRops = {
     hidden?: boolean;
     type?: 'success' | 'warning' | 'info';
     // used as keyName in the config if dismissing the status should be persisted, so it is not
     // shown again. Use an empty string if it should be dismissable without storing it in the
     // config, so the status will be shown again the next time.
     dismissable?: string;
-    className?: string;
+  className?: string;
+  children: ReactNode;
 }
 
-type Props = StatusProps;
-
-export default class Status extends Component<Props, State> {
-  public readonly state: State = {
-    show: true,
-  };
-
-  public componentDidMount() {
-    this.checkConfig();
-  }
+const Status = ({ hidden, type = 'warning', dismissable, className, children }: TPRops) => {
```

Comment:
tiny nit: do you mind adding a newline for every prop, since improves readability